contribution/development_guide.md

```diff
@@ -176,19 +174,19 @@
      LSM - AppArmor
      ```
 
-     KubeArmor is designed for Kubernetes, which means that Kubernetes should be ready in your environment. If Kubernetes is not prepared yet, please refer to [Kubernetes installation guide](self-managed-k8s/README.md). KubeArmor also requires either Docker or Containerd since it internally uses its APIs. KubeArmor requires LSMs to operate properly; thus, please make sure that your environment supports LSMs \(at least, AppArmor\). Otherwise, KubeArmor will work as Audit-Mode with no container behavior restriction.
-
-      * Alternative Setup - Minikube
-
-        MiniKube does not support LSMs by default; thus, we provide the custom ISO image for testing KubeArmor on Minikube.  
-        Please follow the instructions in [Minikube installation guide](minikube/README.md).
+     KubeArmor is designed for Kubernetes, which means that Kubernetes should be ready in your environment. If Kubernetes is not prepared yet, please refer to [Kubernetes installation guide](self-managed-k8s/README.md).
+     KubeArmor requires either Docker or Containerd since it internally uses their APIs. KubeArmor also requires LSMs to operate properly; thus, please make sure that your environment supports LSMs \(at least AppArmor\). Otherwise, KubeArmor will work in Audit-Mode with no container behavior restriction.
```

Comment:
```suggestion
     KubeArmor leverages CRI (Container Runtime Interfaces) APIs and works with Docker or Containerd or CRIO based container runtimes. KubeArmor uses LSMs for policy enforcement; thus, please make sure that your environment supports LSMs \(either AppArmor or bpf-lsm\). Otherwise, KubeArmor will operate in Audit-Mode with no policy "enforcement" support.
```